[Kodi] missing requirement [org.openhab.binding.kodi/2.3.0.201803022259] osgi.wiring.package

Since snapshot build 1221, OH will not load any bindings if the Kodi binding is installed, and reports the following error...

`2018-03-04 21:40:12.931 [ERROR] [org.openhab.core.karaf.internal.FeatureInstaller  ] - Failed installing 'openhab-persistence-jdbc-mariadb, openhab-misc-restdocs, openhab-misc-openhabcloud, openhab-transformation-javascript, openhab-binding-network, openhab-binding-caldav-command1, openhab-ui-habpanel, openhab-transformation-regex, openhab-misc-hueemulation, openhab-binding-weatherunderground, openhab-transformation-jsonpath, openhab-binding-mqtt1, openhab-binding-zoneminder, openhab-binding-wifiled, openhab-transformation-map, openhab-ui-classic, openhab-action-mail, openhab-binding-kodi, openhab-binding-caldav-personal1, openhab-binding-astro, openhab-ui-habmin, openhab-ui-paper': Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-binding-kodi; type=karaf.feature; version="[2.3.0.SNAPSHOT,2.3.0.SNAPSHOT]"; filter:="(&(osgi.identity=openhab-binding-kodi)(type=karaf.feature)(version>=2.3.0.SNAPSHOT)(version<=2.3.0.SNAPSHOT))" [caused by: Unable to resolve openhab-binding-kodi/2.3.0.SNAPSHOT: missing requirement [openhab-binding-kodi/2.3.0.SNAPSHOT] osgi.identity; osgi.identity=org.openhab.binding.kodi; type=osgi.bundle; version="[2.3.0.201803022259,2.3.0.201803022259]"; resolution:=mandatory [caused by: Unable to resolve org.openhab.binding.kodi/2.3.0.201803022259: missing requirement [org.openhab.binding.kodi/2.3.0.201803022259] osgi.wiring.package; filter:="(osgi.wiring.package=org.osgi.service.component.annotations)"]]`

This is affecting others, [as reported in the forum](https://community.openhab.org/t/after-oh2-3-upgrade-fail-to-install-any-features-error-core-karaf-internal-featureinstaller-failed-installing-openhab/41209).

--- a/addons/binding/org.openhab.binding.kodi/META-INF/MANIFEST.MF
+++ b/addons/binding/org.openhab.binding.kodi/META-INF/MANIFEST.MF
@@ -37,6 +37,5 @@ Import-Package:
  org.openhab.binding.kodi.handler,
  org.osgi.framework,
  org.osgi.service.component,
- org.osgi.service.component.annotations,
  org.slf4j
 Service-Component: OSGI-INF/*.xml

